File: exercises/python/pangram/pangram.py
def is_pangram(sentence = ""):
    # list of all letters
    all_letters = "abcdefghijklmnopqrstuvwxyz"

    # iterate through all_letters
    # if a letter is not found in sentence, return False
    for char in all_letters:
        if sentence.lower().find(char) == -1:
            return False

    return True

File: exercises/ruby/pangram/pangram.rb
module Pangram
  def self.pangram?(sentence)
    letters = ('A'..'Z').to_a

    # Iterate through each ASCII letter and check if letter is in sentence
    letters.each do |letter|
      return false unless sentence.upcase.include?(letter)
    end

    # If all letter were found, return true
    true
  end
end
